Subject: Replace generic man(1) instructions with Debian-specific information.

Indicate that the user needs to install the perl-doc package.


---
 pod/perl.pod |   12 ++++++++++--
 1 files changed, 10 insertions(+), 2 deletions(-)

diff --git a/pod/perl.pod b/pod/perl.pod
index e48e526..3b0bbf6 100644
--- a/pod/perl.pod
+++ b/pod/perl.pod
@@ -241,8 +241,16 @@ For ease of access, the Perl manual has been split up into several sections.
     perlwin32		Perl notes for Windows
 
 
-On a Unix-like system, these documentation files will usually also be
-available as manpages for use with the F<man> program.
+On Debian systems, you need to install the B<perl-doc> package which
+contains the majority of the standard Perl documentation and the
+F<perldoc> program.
+
+Extensive additional documentation for Perl modules is available, both
+those distributed with Perl and third-party modules which are packaged
+or locally installed.
+
+You should be able to view Perl's documentation with your man(1)
+program or perldoc(1).
 
 In general, if something strange has gone wrong with your program and you're
 not sure where you should look for help, try the B<-w> switch first.  It will
-- 
tg: (c823880..) debian/doc_info (depends on: upstream)
